core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -12,524 +12,325 @@
-            browser.wait(EC.elementToBeClickable(jazzServices_po.getCreateService()), timeOutHigh);
-            winhandle = browser.getWindowHandle();
-            //To create Service-API
-            jazzServices_po.getCreateService().click();
-            var min = 111111111;
-            var max = 999999999;
-            var randomNum = Math.floor(Math.random() * (max - min + 1)) + min;
-            servicename='servicename' + randomNum;
-            createservice(servicename);
-            serviceapprover();
-            browser.driver.sleep(15000);
-            //Assert-Verifying the created service,Type and Status of the API
-            expect(jazzServices_po.getAwsServiceName().getText()).toEqual(servicename);
-            expect(jazzServices_po.getAPIType().getText()).toEqual('api');
-            expect(jazzServices_po.getAPIStatus().getText()).toEqual('creation started');
-            browser.driver.sleep(100000);
-            browser.wait(EC.visibilityOf(jazzServices_po.serviceStatus()), timeOutHigh);
-            expect(jazzServices_po.getAPIStatus().getText()).toEqual('active');
-      });
+  });
 
-      it('Verify API Service and Navigation', () => {            
-            browser.wait(EC.visibilityOf(jazzServices_po.getAwsServiceName()), timeOutHigh);
-            browser.wait(EC.elementToBeClickable(jazzServices_po.getAwsServiceName()), timeOutHigh);
-            //To Navigate to the particular service and verifying the Page
-            jazzServices_po.getAwsServiceName().click();
-            expect(jazzServices_po.getOverviewStatus().getText()).toEqual('OVERVIEW');
-            browser.wait(EC.visibilityOf(jazzServices_po.getServiceNameHeader()), timeOutHigh);
-            browser.sleep(10000);
-            browser.wait(EC.elementToBeClickable(jazzServices_po.getRefresh()), timeOutHigh);
-            jazzServices_po.getRefresh().click();
-            browser.sleep(5000);
-            browser.wait(EC.visibilityOf(jazzServices_po.getProdName()), timeOutHigh);
-            //To get the corresponding environment[Prod]
-            jazzServices_po.getProdName().click();
-            //Verifying the browser id at the Deployment Tab
-            browser.sleep(5000);
-            expect(jazzServices_po.getDeploymentStatus().getText()).toEqual('DEPLOYMENTS');
-            browser.wait(EC.visibilityOf(jazzServices_po.getProdHeader()), timeOutHigh);
-            browser.wait(EC.visibilityOf(jazzServices_po.getRefresh()), timeOutHigh);
-            browser.driver.switchTo().activeElement();
-            browser.sleep(10000);
-      });
+  function createservice(servicename) {
+    jazzServices_po.getServiceName().sendKeys(servicename);
+    jazzServices_po.getNameSpace().sendKeys('jazztest');
+    jazzServices_po.getServiceDescription().sendKeys('Testing');
+  }
+  function serviceapprover() {
+    browser.driver.sleep(5000);
+    jazzServices_po.getSubmit().click();
+    fluentwaittry(jazzServices_po.getDone(), 15000);
```

Comment:
Across all scripts:
1. Use of params that can be changed easily in one place instead of using hard coded values everywhere.
2. Use of functions for common code used in different scripts.

Reply:
This file's been deleted